src/components/displaySvg/finalSvgDisplay.scss

```diff
@@ -0,0 +1,69 @@
+.display-page-div{
+    padding: 171px;
+    width: 373px; 
+}
+
+  a:hover {
+   background-color: yellow;
+   fill:yellow;
+ }
+
+ svg{
+    max-height: 106px;
+    max-width: fit-content;
+    margin-top: 94px;
```

Comment:
this would make your site unresponsive and affect other components more than its own, preferably use padding